.github/workflows/c_lint.yaml

```diff
@@ -6,6 +6,14 @@ on:
       - "**.c"
       - "**.cpp"
       - "**.h"
+  pull_request:
+    paths:
+      # only when a .c, .cpp, or .h file is changed
+      - "**.c"
+      - "**.cpp"
+      - "**.h"
+  workflow_dispatch:
```

Comment:
Could there be an issue in the Git history of this branch that caused this to appear as a new change? These are changes from #83 and should probably not show up here.